### ISSUE TITLE
Broken link in README.md

### ISSUE THREAD

**Issue by koepalex** [OpenManufacturingPlatform/rules_of_engagement#3]
The link [chartered](https://github.com/OpenManufacturingPlatform/rules_of_engagement/blob/staging) is pointing to non existing page.

**Comment by jpradocueva** [comment]
Yes, I am trying to figure out where we should store the WG charters.
Charters should be public therefore, we need to decide where to add them, here in this repo or in e.g. https://github.com/OpenManufacturingPlatform/openmanufacturingplatform.github.io
